### DISCUSSION TITLE
Why `model.cdf()` returns different results from `model.distribution.distribution.cdf()`?

### DISCUSSION THREAD

**Discussion by nucflash** [georgebv/pyextremes#89] [Q&A] [answered]
I'm experimenting with distributions, and I found out that `model.cdf(x)` returns different results from `model.distribution.distribution.cdf(x)`. Why is that, and which one should I use?

**Comment by georgebv** [comment, marked as answer]
This is because `model` uses transformation internally while `distribution` doesn't, see this answer https://github.com/georgebv/pyextremes/discussions/90#discussioncomment-10232510

**Reply by nucflash** [reply]
Thank you!
